packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -428,36 +413,32 @@ export default class ImageEdit implements EditorPlugin {
             .getComputedStyle(this.image)
             .getPropertyValue(property);
     }
-
-    /**
-     * Get image wrapper from image
-     */
-    private getImageWrapper(): HTMLElement {
-        return document.getElementById(this.imageId);
-    }
-
     /**
      * Remove the temp wrapper of the image
      */
     private removeWrapper = () => {
-        const wrapperImage = this.getImageWrapper();
         const doc = this.editor.getDocument();
-        doc.body?.removeChild(wrapperImage);
-        this.image.style.removeProperty('visibility');
+        if (this.wrapper && doc.body?.contains(this.wrapper)) {
+            doc.body?.removeChild(this.wrapper);
+        }
+        this.image.className = '';
+        this.wrapper = null;
     };
 
     /**
      * Update image edit elements to reflect current editing result
      * @param context
      */
     private updateWrapper = (context?: DragAndDropContext) => {
-        const wrapper = this.getImageWrapper();
-        if (wrapper) {
+        if (this.wrapper) {
```

Comment:
consider do
```ts
const wrapper = this.wrapper;

if (wrapper) { 
...
```

So no need to do use "this.wrapper" everywhere to save some bytes